src/In.ProjectEKA.HipService/DataFlow/DataEntryFactory.cs

```diff
@@ -76,29 +77,29 @@ public DataEntryFactory()
             return Option.Some(new EncryptedEntries(processedEntries.AsEnumerable(), keyMaterial));
         }
 
-        private Entry StoreComponentAndGetLink(Entry componentEntry, string transactionId)
+        private Entry StoreComponentAndGetLink(Entry componentEntry, string careContextReference)
```

Comment:
we dont need transactionid is it?

Reply:
Yes, we were not using it there. That's why I removed. Because of this Codacy was failed before